src/backend/joanie/core/serializers/client.py

```diff
@@ -720,6 +722,14 @@ def get_target_enrollments(self, order) -> list[dict]:
             context=self.context,
         ).data
 
+    def validate_has_consent_to_terms(self, value):
+        """Check that user has accepted terms and conditions."""
+        if value is False:
+            message = _("You must accept the terms and conditions to proceed.")
+            raise serializers.ValidationError({"has_consent_to_terms": [message]})
```

Comment:
You juste have to raise a `ValidationError` with a message here I think. Because you are using the `validate_has_consent_to_terms` method, DRF will do that job for you.

Reply:
Indeed !